sphinx-doc/deprecated.rst

```diff
@@ -77,6 +77,10 @@ Commands and features deprecated in v2.x will be removed in v3.0.
      - Replace with
    * - ``group`` parameter to ``integrate.mode_minimize_fire``
      - Pass group to integration method.
+   * - ``alpha`` parameter to ``pair.lj`` and related classes
+     - n/a
+   * - LJ 12-8 pair potential
+     - Mie potential
```

Comment:
Here you can also mention the active forces: the array of tuples f_list and t_list have been replaced with the tuples active_force and active_torque

Reply:
done